src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,83 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+
+  h1 {
+    color: $c-text-primary-white;
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+}
+
+.recommender-grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender__grid__material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  .button-favourite {
+    cursor: pointer;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+
+    .icon-favourite {
+      color: $c-text-primary-white;
+      stroke: $c-text-primary-white;
+      fill: none;
+
+      &--filled {
+        stroke: $c-text-primary-white;
+        color: $c-text-primary-white;
+        fill: $c-text-primary-white;
+      }
+    }
+  }
+
+  .cover-container {
+    justify-content: center;
+    width: 100%;
+  }
```

Comment:
See comment above regarding nesting of elements.

[The search result item uses a wrapper div for positioning](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/develop/src/stories/Library/search-result-item/SearchResultItem.tsx#L26).

Reply:
I'll fix